template-manager-cmtemplate/src/test/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/tez/TezRoleConfigProviderTest.java

```diff
@@ -1,12 +1,16 @@
 package com.sequenceiq.cloudbreak.cmtemplate.configproviders.tez;
 
+import static com.sequenceiq.cloudbreak.cmtemplate.configproviders.core.CoreConfigProvider.CORE_DEFAULTFS;
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
 
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
 
+import com.sequenceiq.cloudbreak.cloud.model.ClouderaManagerProduct;
+import com.sequenceiq.cloudbreak.cloud.model.ClouderaManagerRepo;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.mockito.junit.MockitoJUnitRunner;
```

Comment:
I know most legacy unit tests are still based on JUnit 4, but we shall gradually migrate them to JUnit 5 if possible. Could you please use the new Jupiter types here? See `HiveKnoxConfigProviderTest` as an example.

Reply:
Addressed!